### PR TITLE
Fabric Website Github FeedbackList: Iterate Github issues collection directly to avoid possible undefined value

### LINKED ISSUE #8276: Github Issues Feature on Component Pages not working
### Environment Information 
 
- __Package version(s)__: n/a
- __Browser and OS versions__: n/a

### Please provide a reproduction of the bug in a codepen:
 
https://developer.microsoft.com/en-us/fabric#/components/choicegroup
 
#### Actual behavior:
 
Page is throwing an exception and Github Issues feature is not working. Issue seems to be in `FeedbackList` attempting to use `created_at` property of Github query results.

image

 
#### Expected behavior:
 
Github Iussue list feature appears.

#### Priorities and help requested:
 
Are you willing to submit a PR to fix? (Yes, No)
 
Requested priority: (Blocking, High, Normal, Low) 
 
Products/sites affected: (if applicable)

### PR DESCRIPTION
**Pull request by KevinTCoughlin**
#### Pull request checklist

- [x] Addresses an existing issue: Fixes #8276
- [x] Include a change request file using `$ npm run change`

#### Description of changes

`FeedbackList` was using the API response value of `total_count` as its iterator max which doesn't necessarily map to the length of the collection being iterated over. Because of this, index lookup would sometimes return `undefined` which would result in a runtime exception.

The fix is to simply iterate over the issues collection for the response directly.

#### Focus areas to test

- CI should pass
- Build should pass
- Github issues widget should render properly on Fabric Website

**ChoiceGroup page example**:

image

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/8280)

### CODE REVIEW

**Comment by KevinTCoughlin** [comment]
On the ChoiceGroup example page, we were receiving `total_count: 33`, however the API response only returned `30`. Therefore we would encounter `undefined`. IIRC the Github API response batches by `30` by default... so as the issue count grows per label this problem would surface across various pages.

File: packages/example-app-base/src/components/FeedbackList/FeedbackList.tsx [hunk 1]
```diff
@@ -46,22 +46,22 @@ export class FeedbackList extends React.Component<IFeedbackListProps, IFeedbackL
   }
 
   public async getIssues(url: string): Promise<IListItem[]> {
-    let issueList: IListItem[] = [];
-
     const response = await fetch(url);
```
Comment (cliffkoh) [inline packages/example-app-base/src/components/FeedbackList/FeedbackList.tsx]: this entire routine is a little buggy in that it does not handle pagination... (it simply assumes no pagination takes place)

**Comment by KevinTCoughlin** [comment]
> Can we just slightly augment the UI that it is showing 30 most recent issues, if issues total_count >30?
> 
> If not, could you minimally add comments stating that the current code intentionally shows only the first 30 and handling pagination is future work?

I'll handle pagination as part of this PR within the next day as adding comments to the existing code is roughly the same amount of work (design questions aside).

File: packages/example-app-base/src/components/FeedbackList/FeedbackList.tsx (hunk 1 above)
Comment (KevinTCoughlin) [reply packages/example-app-base/src/components/FeedbackList/FeedbackList.tsx]: At least it doesn't throw a runtime exception AND not handle pagination tho ;)

**Pull request merged**